test/lint/test-devdeps.js

```diff
@@ -0,0 +1,46 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+import expect from 'expect';
+import fs from 'fs-promise';
+import without from 'lodash/without';
+import {
+  REQUIRES_REGEX,
+  IMPORTS_REGEX,
+  autoFailingAsyncTest,
+  globMany,
+  regexFiles,
+} from './shared.js';
+
+const all = '/**/';
+const valid = '*.@(js|jsx)';
+
+const paths = [
+  `app${all}${valid}`,
+];
+
+// XXX: Which of these do we actually need to ignore?
+const ignored = [
+  'electron-ipc-mock',
+
+  // XXX: I'm not sure about these ones...
+  'express',
+  'react-addons-perf',
+  'source-map-support',
+  'uuid',
```

Comment:
react-addons-perf can be ignored. The others should be put into dependencies, no devDependencies (looks like express is in both)